app/assets/javascripts/discourse/app/components/modal/bulk-topic-actions.gjs

```diff
@@ -165,6 +179,14 @@ export default class BulkTopicActions extends Component {
           (t) => t.set("category_id", this.categoryId)
         );
         break;
+      default:
+        // Plugins can register their own custom actions via onRegisterAction
+        // when the activeComponent is rendered.
+        if (this.customAction) {
+          this.customAction(this.performAndRefresh.bind(this));
+        } else {
+          _customActions[this.args.model.initialActionLabel](this);
```

Comment:
Is all this `_customActions` stuff just backwards-compat for the old way?

Reply:
Ya, I'm open to suggestions, but it was to get `unassign` to work, not necessarily backwards compatibility.

```
      api.addBulkActionButton({
        label: "topics.bulk.unassign",
        icon: "user-times",
        class: "btn-default unassign-topics",
        action({ performAndRefresh }) {
          performAndRefresh({ type: "unassign" });
        },
        actionType: 'performAndRefresh',
      });
```

Since it doesn't have a component with a registerAction call.